github-sync/github-data/users.yaml

```diff
@@ -224,6 +224,10 @@ users:
         role: member
       - name: policy-controller-codeowners
         role: member
+      - name: public-good-instance-team
+        role: maintainer
```

Comment:
Instead of this, you'll need to add yourself with `maintain` as a collaborator - https://github.com/sigstore/community/blob/53e5b6494927670901885dc870be7ebbbbec3174/github-sync/github-data/repositories.yaml#L847

This just lets you manage the public-good-instance-team group, which only has triage

Reply:
Done, thanks.